src/cosmosdb-preview/azext_cosmosdb_preview/_help.py

```diff
@@ -7,6 +7,142 @@
 
 from knack.help_files import helps  # pylint: disable=unused-import
 
+helps['cosmosdb mongocluster firewall'] = """
+type: group
+short-summary: Mongo cluster firewall.
+"""
+
+helps['cosmosdb mongocluster firewall rule'] = """
+type: group
+short-summary: Mongo cluster firewall rule.
+"""
```

Comment:
If there's only one sub commnd group under `firewall`, I think you can combine them to `cosmosdb mongocluster firewall-rule` command group

Reply:
We have reviewed the command spec with the PM and its approved. I do not think we should change this in the last minute. We want to be consistient with other APIs like mongodb create **role definition**.

Reply:
There're two sub command groups under `role` which are `role assignment` and `role definition`. But you only have one command group under `firewall`.

But if this is the final design, fine.

Reply:
I think we should keep it this is approved by the PM team